### DISCUSSION TITLE
Assessment of the difficulty in porting CPU architecture for moosefs

### DISCUSSION THREAD

**Discussion by wangyuliu** [moosefs/moosefs#549] [Q&A]
Hello everyone! I am working on implementing a tool to assess the complexity of CPU architecture porting. It primarily focuses on RISC-V architecture porting. As part of my dataset, I have collected moosefs project. I would like to gather community opinions to support my assessment. I appreciate your help and response! Based on scanning tools, the porting complexity is determined to be simple, with a small amount of code related to the CPU architecture in the project. (referring to the overall workload from adapting the project to a specific architecture to achieving full functionality on that architecture). Is this assessment accurate? I look forward to your help and response.

**Comment by chogata** [comment]
There should be zero additional work required to port MooseFS to a RISC machine. MooseFS honours the processor's byte order and has no processor specific code included.
